app/helpers/reactjs_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ReactjsHelper
   def mount_react_component(name, selector, data = [], opts = {})
     javascript_tag defer: 'defer' do
-      "$(tfm.reactMounter.mount('#{name}', '#{selector}', #{data}, #{opts[:flatten_data] || false}));".html_safe
+      "$(tfm.reactMounter.mount('#{name}', '#{selector}', #{data}, #{app_metadata.to_json}, #{opts[:flatten_data] || false}, #{opts[:context]}));".html_safe
```

Comment:
Wouldn't it break current usages of `mount_react_component` `options`?

Reply:
I have change `mount` function, but it doesn't affect the options argument of `mount_react_component`